components/drawer/style/drawer.less

```diff
@@ -191,13 +191,27 @@
     background: @component-background;
   }
 
+  &-wrapper-body {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: nowrap;
+    width: 100%;
+    height: 100%;
+  }
+
   &-body {
+    flex-grow: 1;
     padding: @drawer-body-padding;
+    overflow: auto;
     font-size: @font-size-base;
     line-height: @line-height-base;
     word-wrap: break-word;
   }
 
+  &-footer {
+    flex-shrink: 0;
```

Comment:
add height and border-top,, ref: https://github.com/ant-design/ant-design/blob/master/components/modal/style/modal.less#L97-L107